workloads/keyvault/README.md

```diff
@@ -0,0 +1,58 @@
+# Azure Key Vault
+
+## Overview
+
+The ARM template and Bicep file for Azure Key Vault is developed for organizations to accelerate their deployment while ensuring the Azure Service is compliant and meets your organization's requirements for hardening PaaS services.
+
+## Compliant Azure Key Vault
+
+It is assumed that the deployment will go into a landing zone where the platform provides the guardrails (Azure Policy), such as:
+
+- Diagnostics and metrics are enabled to route security relevant information to a platform Log Analytics workspace (this does not prevent application teams to also use their app centric Log Analytics workspace, which also will have a diagnostic setting configured to send logs/metrics.)
+- Usage of public endpoint is not allowed for PaaS services
+- Azure Defender (Azure Security Center) is enabled for Azure Key Vault in the landing zones
```

Comment:
, and private endpoints DNS records are automatically created in the privatelink.vaultcore.azure.net Azure Private DNS zone in the connectivity subscription.